action.yml

```diff
@@ -12,7 +12,10 @@ inputs:
     required: true
   issue_identifiers:
     description: 'One or more issues to update, for example see ./github/workflows/ci.yml'
-    required: true
+    required: false
```

Comment:
By design it was required in the first place, but after adding one more way to filter issue it becomes optional.

Since we are checking that at least one option is provided and throwing an error if not -- we are safe